tests/model_registry/utils.py

```
@@ -795,3 +799,91 @@ def wait_for_default_resource_cleanedup(admin_client: DynamicClient, namespace_n
+        raise e
+
+
+def get_byoidc_issuer_url(admin_client: DynamicClient) -> str:
+    authentication = get_cluster_authentication(admin_client=admin_client)
+    assert authentication is not None
+    url = authentication.instance.spec.oidcProviders[0].issuer.issuerURL
+    assert url is not None
+    return url
+
+
+def get_byoidc_user_credentials(username: str = None) -> Dict[str, str]:
```

Ruff (0.14.5)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)